### PR TITLE
Rendering fewer ticks on bar chart when width is less than 400px

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard13**
## Summary

Rendering fewer ticks on the bar chart on mobile so there is no tick label overlap

## Detailed design

Conditionally setting the number of ticks in `generateBarChartCoordinates` based on the width

### CODE REVIEW

**Review by VWSCoronaDashboard4 — APPROVED** [review]

File: packages/app/src/components-styled/bar-chart/bar-chart-coordinates.ts [hunk 1]
```diff
@@ -61,7 +61,7 @@ function generateBarChartCoordinates(
   const barsHeight = values.length * 35;
   const height = barsHeight + spacing.top + spacing.bottom;
 
-  const numTicks = 10;
+  const numTicks = width > 400 ? 10 : 6;
```
Comment (VWSCoronaDashboard1) [inline packages/app/src/components-styled/bar-chart/bar-chart-coordinates.ts:64]: In this PR: https://github.com/minvws/nl-covid19-data-dashboard/pull/2154/files#diff-4af1f3aab2c05a3d786e2f4e6277c9acd28014b03cea47364c1fd97489f705ca
I've added the `useBreakpoints()` hook, its probably better to determine this tick number based on one of the breakpoints there in. In this case `breakpoints.xs`.

**Review by VWSCoronaDashboard1 — CHANGES_REQUESTED** [review]

File: packages/app/src/components-styled/bar-chart/bar-chart-coordinates.ts (hunk 1 above)
Comment (VWSCoronaDashboard13) [reply packages/app/src/components-styled/bar-chart/bar-chart-coordinates.ts:64]: Alright, I'll wait for that PR to get merged in before implementing this update

Comment (VWSCoronaDashboard13) [reply packages/app/src/components-styled/bar-chart/bar-chart-coordinates.ts:64]: Actually I just read another thread on slack discussing this issue where the consensus was to keep the existing implementation!

Comment (VWSCoronaDashboard10) [reply packages/app/src/components-styled/bar-chart/bar-chart-coordinates.ts:64]: Nope, your implementation is the correnct one indeed, we can merge this 👍

**Review by VWSCoronaDashboard10 — APPROVED** [review]

**Pull request merged**